recipes/pyscamp-gpu/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pyscamp" %}
+{% set version = "3.0.2" %}
+
+
+package:
+  name: {{ name|lower }}-gpu
+  version: {{ version }}
+
+source:
+  # Temporarily use test pypi package to test new code.
+  url: https://test-files.pythonhosted.org/packages/8e/cc/4216ddadb2b7c2d008802e7c7b1c32d4483f6ccb6fb320a956a368fffef9/pyscamp-3.0.3.dev4.tar.gz
+  sha256: 80081eab782d96111151a50d6924aafb617568cb94fe81a8da304095cd286fcf
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyscamp-{{ version }}.tar.gz
+  #sha256: 1a9dcae45c85d993910ecb75bbb28188e72cf8f0c086c876ef73d145793d8060
+
+build:
+  number: 0
+  skip: True  # [cuda_compiler_version == "None"]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [(win64 or linux64) and cuda_compiler_version != "None"]
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+    - pybind11 2.9.2
+    - eigen 3.4.0
```

Comment:
You want these in host, not build AFAIK.

Reply:
I was thinking that too, but since they were header only I put them in build. I guess it is good to be safe here and make sure there's not a chance the user will be missing something they need.